console/src/components/Services/RemoteSchema/RemoteSchemaSubSidebar.js

```diff
@@ -46,34 +52,55 @@ const RemoteSchemaSubSidebar = ({
         </li>
       );
     } else {
-      childList = _dataList.map((d, i) => {
-        let activeTableClass = '';
-        if (
-          d.name === viewRemoteSchema &&
-          location.pathname.includes(viewRemoteSchema)
-        ) {
-          activeTableClass = styles.activeLink;
-        }
+      if (_dataList.length > 0) {
+        childList = _dataList.map((d, i) => {
+          let activeTableClass = '';
+          let isSchemaInconsistent = false;
+          let inconsistentCurrentSchema;
+          const filteredList = inconsistentRemoteSchemas.filter(
+            inconObject => inconObject.definition.name === d.name
+          );
+
+          if (
+            d.name === viewRemoteSchema &&
+            location.pathname.includes(viewRemoteSchema)
+          ) {
+            activeTableClass = styles.activeLink;
+          }
+
+          if (filteredList.length) {
+            isSchemaInconsistent = true;
+            inconsistentCurrentSchema = filteredList[0];
+          }
```

Comment:
You can use `find` instead of filter. 

Also, let's use only one variable for this:

```suggestion
          let activeTableClass = '';
          if (
            d.name === viewRemoteSchema &&
            location.pathname.includes(viewRemoteSchema)
          ) {
            activeTableClass = styles.activeLink;
          }

          const inconsistentSchema = inconsistentRemoteSchemas.find(elem => elem.definition.name === d.name)
```

And then use `inconsistentSchema` to do:

```js
   {inconsistentSchema ? (
                  <ToolTip message={inconsistentSchema.reason}>
                     // ....
                  </ToolTip>
                ) : null}
```